monash-human-power/MHP_DAS
# install mqtt-logger package

## Description

This PR installs `mqtt-logger` from PyPi instead of using a local version. This is meant to simplify testing and development of the logger.

## Steps to Test

Check that `poetry run python -m das.V3_mqtt_recorder /v3/#  -v --host broker.hivemq.com` runs.
